storage/filesystem/storage.go

```diff
@@ -11,6 +12,8 @@ import (
 // standard git format (this is, the .git directory). Zero values of this type
 // are not safe to use, see the NewStorage function below.
 type Storage struct {
+	storer.Options
```

Comment:
Why is an embedded struct? and not a private field?

Reply:
You're right. There's no reason why it should be embedded or public. It's now changed.